test/shared/configUtils.test.ts

```diff
@@ -5,74 +5,89 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
```

Comment:
Same here... let's limit these to only the lines that need them instead of the entire file